src/components/Book.js

```diff
@@ -1,29 +1,85 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
 import { removeBook, deleteBook } from '../redux/features/books/booksSlice';
 
+import 'react-circular-progressbar/dist/styles.css';
+
 function Book({
   itemId, title, author, category,
 }) {
+  const [chapterNumber, setChapterNumber] = useState({
+    chapter: 0,
+    percent: 0,
+  });
   const dispatch = useDispatch();
 
   const remover = (itemId) => {
     dispatch(deleteBook(itemId)).then(() => dispatch(removeBook(itemId)));
   };
   return (
-    <div>
-      <p>{category}</p>
-      <h2>{title}</h2>
-      <p>{author}</p>
+    <div className="book-container">
+      <div className="book-item-main">
+        <p className="category">{category}</p>
+        <h2>{title}</h2>
+        <p className="author">{author}</p>
+        <button type="submit" className="button-show">
+          Comments
+        </button>
+        <button
+          type="submit"
+          className="button-show remove"
+          onClick={() => remover(itemId)}
+          z
+        >
+          Remove
+        </button>
+        <button type="submit" className="button-show">
+          Edit
+        </button>
+      </div>
+      <div style={{ display: 'flex', alignItems: 'center' }}>
+        <span>
+          {chapterNumber.percent < 100 ? (
+            <CircularProgressbar
+              className="circular-progressbar"
+              value={chapterNumber.percent}
+              text={`${chapterNumber.percent} %`}
+            />
+          ) : (
+            <CircularProgressbar
+              className="circular-progressbar"
+              value={100}
+              text={`${100} %`}
+            />
+          )}
+        </span>
+        <span style={{ marginLeft: '1rem' }}>Completed</span>
+      </div>
```

Comment:
- [ ] Well done :clap: your implementation is very good but as I mentioned above :point_up_2: please follow [Zaplin](https://zeplin.io/) design but your `progress bar` is not the same as [Zaplin](https://zeplin.io/) design so kindly stick to the [Zaplin](https://zeplin.io/) design :+1:  check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

| Your Design | Zaplin Design|
|--------|--------|
  | ![Screenshot from 2023-05-13 16-00-37](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/3eece6bf-2982-4b6a-a982-3c6cd251920b) | ![Screenshot from 2023-05-13 16-00-22](https://github.com/Lul-Abdifan/Bookstore/assets/45226848/4ae10e14-4b2f-4e11-a3b1-b1ad19205674) |